### PR TITLE
updating for new domain

### PR DESCRIPTION
**Pull request by LilaHickey**
@malev all minor - text replacement, no actual code changing.

### CODE REVIEW

File: binstar_build_client/commands/submit.py [hunk 1]
```diff
@@ -106,7 +106,7 @@ def print_build_results(args, build):
     log.info('')
     build_result_url = build.get('url')
     if not build_result_url:
-        build_result_url = 'http://alpha.binstar.org/%s/%s/builds/matrix/%s' % (args.package.user, args.package.name, build['build_no'])
+        build_result_url = 'http://alpha.anaconda.org/%s/%s/builds/matrix/%s' % (args.package.user, args.package.name, build['build_no'])
```
Comment (malev) [inline binstar_build_client/commands/submit.py]: we don't have a `alpha.anaconda.org` server yet :cry:

File: binstar_build_client/commands/trigger.py [hunk 2]
```diff
@@ -28,7 +28,7 @@ def main(args):
                                filter_platform=args.platform)
 
     log.info('')
-    log.info('To view this build go to http://alpha.binstar.org/%s/%s/builds/matrix/%s' % (args.package.user, args.package.name, build_no))
+    log.info('To view this build go to http://alpha.anaconda.org/%s/%s/builds/matrix/%s' % (args.package.user, args.package.name, build_no))
```
Comment (malev) [inline binstar_build_client/commands/trigger.py]: Same here

**Comment by malev** [comment]
Why is the build broken?

**Comment by LilaHickey** [comment]
@malev - updated to just referenced http://anaconda.org, since that will show the users' builds, and does currently exist :D

File: binstar_build_client/commands/trigger.py [hunk 3]
```diff
@@ -28,7 +28,7 @@ def main(args):
                                filter_platform=args.platform)
 
     log.info('')
-    log.info('To view this build go to http://alpha.binstar.org/%s/%s/builds/matrix/%s' % (args.package.user, args.package.name, build_no))
+    log.info('To view this build go to http://anaconda.org/%s/%s/builds/matrix/%s' % (args.package.user, args.package.name, build_no))
```
Comment (malev) [inline binstar_build_client/commands/trigger.py]: it used to say alpha.binstar.org but it does not makes too much sense

**Comment by malev** [comment]
LGTM, I have only one question/concern and it's about `alpha.binstar.org` which I think it just wasn't correct. What do you think @LilaHickey ?

**Pull request merged**